app/src/main/java/org/fossasia/openevent/general/event/topic/SimilarEventsViewModel.kt

```diff
@@ -17,32 +17,54 @@ class SimilarEventsViewModel(private val eventService: EventService) : ViewModel
 
     private val mutableProgress = MutableLiveData<Boolean>()
     val progress: LiveData<Boolean> = mutableProgress
-    private val mutableSimilarEvents = MutableLiveData<List<Event>>()
-    val similarEvents: LiveData<List<Event>> = mutableSimilarEvents
+    private val mutableSimilarLocationEvents = MutableLiveData<List<Event>>()
+    val similarLocationEvents: LiveData<List<Event>> = mutableSimilarLocationEvents
+    private val mutableSimilarIdEvents = MutableLiveData<List<Event>>()
+    val similarIdEvents: LiveData<List<Event>> = mutableSimilarIdEvents
     private val mutableError = SingleLiveEvent<String>()
     val error: LiveData<String> = mutableError
 
     var eventId: Long = -1
 
-    fun loadSimilarEvents(id: Long) {
+    fun loadIdEvents(id: Long) {
```

Comment:
Maybe refactor this to `loadSimilarIdEvents` as all other variables also consist of `similarIdEvents` such as the mutable variable

Reply:
okay, makes sense.

Reply:
resolved